packages/patternfly-4/react-core/src/components/Select/Select.js

```diff
@@ -42,7 +42,6 @@ const defaultProps = {
   selections: null,
   placeholderText: null,
   variant: SelectVariant.single,
-  width: '100%'
```

Comment:
don't we need a default prop here since the width is not required?

Reply:
If I understood correctly, the `width` should be set by `pf-c-select`.
I didn't remove `width` from props in case someone wants to change that attribute, though.

Reply:
Probably makes more sense to default it to empty string or null, this way your check still works below